src/oauth/signature.cr

```diff
@@ -80,7 +80,7 @@ struct OAuth::Signature
     body = request.body
     content_type = request.headers["Content-type"]?
     if body && content_type == "application/x-www-form-urlencoded"
-      params.add_query body
+      params.add_query body.to_s
```

Comment:
Oh my bad, I thought `to_s` returned the entire content as a String but this seems to be only the case for MemoryIO and not File etc. I believe the most intuitive thing is that we read it without raising an error.